samples/Mvc.ActionConstraintSample.Web/Controllers/ItemsController.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Mvc;
+
+namespace ActionConstraintSample.Web.Controllers
```

Comment:
Why group the projects at all? Renaming to this convention seems like you've already done extra work for no gain. In fact extra work for lost directory / namespace consistency -- a consistency we strive for.